library/src/main/scala/scala/scalajs/js/Tuple.scala

```diff
@@ -0,0 +1,671 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js API               **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-lang.org/     **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+
+/**
+ * All doc-comments marked as "MDN" are by Mozilla Contributors,
+ * distributed under the Creative Commons Attribution-ShareAlike license from
+ * https://developer.mozilla.org/en-US/docs/Web/Reference/API
+ */
+package scala.scalajs.js
+
+import scala.scalajs.js.annotation.JSName
+import scala.language.implicitConversions
+
+// scalastyle:off line.size.limit
+
+@native
+sealed trait Tuple1[+T1] extends Object {
```

Comment:
On the other side. Due the implicit conversion there is  no need for iterable on js tuple so far.